gulpfile.js

```diff
@@ -10,25 +10,208 @@
  */
 'use strict';
 
-var gulp = require('gulp');
+var _ = require('lodash');
```

Comment:
The file has become too long, we need to look into breaking this. Lets discuss this in a call.

Reply:
Good idea. I have created a new export file for backstopJS task. Now the file is smaller.